app/views/provider/admin/cms/_tab_content_title.html.slim

```diff
@@ -0,0 +1,5 @@
+section class="pf-c-page__main-section pf-m-light"
+  div class="pf-c-content"
+    h1 = header
+    - if body = local_assigns[:body]
```

Comment:
Shouldn't it be like this?

```suggestion
    - if body == local_assigns[:body]
```

Reply:
No, it's an assignment inside a condition. It basically defines `body` and then checks its value. Alternatively I'd do `if local_assigns.key?(:body)`.